openebs/openebs-provisioner.go

```diff
@@ -108,6 +108,7 @@ func (p *openEBSProvisioner) Provision(options controller.VolumeOptions) (*v1.Pe
 		volumeSpec.Metadata.Labels.StorageClass = *className
 	}
 	volumeSpec.Metadata.Labels.Namespace = options.PVC.Namespace
+	volumeSpec.Metadata.Labels.PersistantVolumeClaim = options.PVC.ObjectMeta.Name
```

Comment:
PVC name will have to be passed, when creating the volume via clone operation. Check in function: SnapshotRestore in file: snapshot/pkg/volume/openebs/processor.go